mayastor/src/bdev/dev/nvmx/controller.rs

```diff
@@ -101,6 +117,7 @@ pub struct NvmeController<'a> {
     prchk_flags: u32,
     pub(crate) state: NvmeControllerState,
     inner: Option<NvmeControllerInner<'a>>,
+    reset_active: bool,
```

Comment:
Would reset_active not be more at home as a variant of NvmeControllerState? i.e ::Resetting ?